neurons/validator.py

```diff
@@ -164,8 +165,9 @@ async def filesystem_test_evaluation(self):
             bt.logging.warning(f"No top hotkey available for competition {data_package.competition_id}")
             top_hotkey = None
         
-        # Enable if you want to have results in CSV for debugging purposes
-        # await self.log_results_to_csv(data_package, top_hotkey, models_results)
+        
+        results_file_name = f"{datetime.datetime.now().strftime('%Y-%m-%d_%H-%M-%S')}-{data_package.competition_id}.csv"
+        await self.log_results_to_csv(results_file_name, data_package, top_hotkey, models_results)
```

Comment:
it was previously commented/optional - maybe we should have some flag for that to not pollute dev env every time when running test eval?

Reply:
well, it's just couple of kb file each run, so no worries that they'll run out of space. and it's useful, we can ask validators to zip and send the directory for checkup